collator/src/collator/do_collate/finalize.rs

```diff
@@ -587,7 +653,7 @@ impl Phase<FinalizeState> {
             block: new_block,
             is_key_block: new_block_info.key_block,
             prev_blocks_ids: self.state.prev_shard_data.blocks_ids().clone(),
-            top_shard_blocks_ids: self.state.collation_data.top_shard_blocks_ids.clone(),
+            top_shard_blocks: self.state.collation_data.top_shard_blocks.clone(),
```

Comment:
Why do we need to change this to the `Vec<TopShardBlockInfo>` if further we use only blocks ids [here](https://github.com/broxus/tycho/pull/480/files#diff-e1c4abb5972d10b9f9c9a8593a763efd50a52d5c9dc1cbaf000ba392260cb052R128)